src/layouts/Layout.astro

```diff
@@ -4,48 +4,29 @@ interface Props {
 }
 
 const { title } = Astro.props;
+
+
 ---
 
 <!doctype html>
-<html lang="en">
+<html lang="es">
 	<head>
 		<meta charset="UTF-8" />
 		<meta name="description" content="Astro description" />
 		<meta name="viewport" content="width=device-width" />
-		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Ubuntu:wght@400;700&display=swap">
+		<link rel="icon"  type="image/svg+xml" href="https://avatars.githubusercontent.com/u/127456648?s=200&v=4" />
 		<meta name="generator" content={Astro.generator} />
-		<title>{title}</title>
+		<title>Computer Society</title>
```

Comment:
I think it's not a good idea to hardcode the title here. Please revert this change and add your title as a prop.